run_scripts/README.md

```diff
@@ -0,0 +1,24 @@
+To run scripts on AWS do the following:
+
```

Comment:
Thank you!

Reply:
You wrote these instructions! I never updated them, and from a cursory glance `ray_autoscale.yaml` is almost certainly broken. I never used AWS, nor ray_autoscale.yaml. The regular `requirements.txt` did change a lot while I never updated `requirements_autoscale.txt`.

I don't have access to an AWS cluster, so if you want to see this working I'm afraid you'll have to bring the relevant files up to speed.

Is there a reason you were using the developer version of rllib on autoscale? Why not install it through including ray[rllib] in requirements.txt?

Reply:
Haha go me then! We were implementing some custom stuff so we needed the developer version. It'd be better to stick it in the requirements now.